### PR TITLE
Add filter by user to list applications endpoint

### PR DESCRIPTION
**Pull request by WillGibson**
**Why**  
So the dropdown in the front end works.

### CODE REVIEW

File: components/ApplicationsList/ApplicationsList.jsx [hunk 1]
```diff
@@ -55,7 +55,8 @@ const ApplicationsList = ({
   useEffect(() => {
     const fetchOfficers = async () => {
       try {
-        const { data } = await axios.get('/api/grant-officers');
+        const data = await fetchGrantOfficers();
+        console.log(data);
         setOfficiers(data.grantOfficers.map(({ identifier }) => identifier));
```
Comment (neilkidd) [inline components/ApplicationsList/ApplicationsList.jsx:60]:
```suggestion
        setOfficers(data.grantOfficers.map(({ identifier }) => identifier));
```

File: components/ApplicationsList/ApplicationsList.jsx [hunk 2]
```diff
@@ -55,7 +55,8 @@ const ApplicationsList = ({
...
       } catch {
         setOfficiers(null);
```
Comment (neilkidd) [inline components/ApplicationsList/ApplicationsList.jsx:62]: `setOfficiers` => setOfficers
Small typo - I expect non breaking. So fix at your option

Comment (WillGibson) [reply components/ApplicationsList/ApplicationsList.jsx:62]: Will do with IDE

File: components/ApplicationsList/ApplicationsList.jsx (hunk 1 above)
Comment (WillGibson) [reply components/ApplicationsList/ApplicationsList.jsx:60]: Doing with IDE

**Pull request merged**